packages/roosterjs-editor-plugins/lib/PasteManager/PasteManager.ts

```diff
@@ -76,8 +77,16 @@ export default class PasteManager implements EditorPlugin {
                 let span = this.editor.getDocument().createElement('span');
                 span.innerHTML = normalizeContent(clipboardData.htmlData);
                 convertPastedContentFromWord(span);
-                this.editor.insertNode(span);
-                unwrap(span);
+
+                if (browserData.isChrome) {
+                    // In Chrome, if insert the span node first then do unwrap, the cursor position will be wrong
+                    while (span.firstChild) {
```

Comment:
Why do you use 'while', wouldn't it keep inserting the node?

Reply:
insertNode will cause the node removed from original parent and append into new parent. So it will not get into infinite loop here.